src/operator/nn/moments-inl.h

```diff
@@ -77,7 +77,7 @@ inline bool MomentsType(const nnvm::NodeAttrs& attrs,
 
 struct VarBroadcastKernel {
   template<typename DType>
-  MSHADOW_XINLINE static void Map(int i,
+  MSHADOW_XINLINE static void Map(index_t i,
                                   DType *out,
                                   const DType *data,
                                   const DType *mean,
```

Comment:
can you change index variables inside the kernels to `index_t` from `size_t`

Reply:
changed